dashlord.yml

```
@@ -461,3 +461,11 @@ urls:
     betaId: homologation
     repositories:
       - betagouv/mon-service-securise
+  - url: https://signaux-faibles.beta.gouv.fr
+    category: startup
+    betaId: signaux-faibles
+    tags:
+      - app
+      - production
+    repositories:
+      - https://github.com/signaux-faibles
```

```suggestion
      - signaux-faibles/signauxfaibles-web
```

Pour l'instant on ne gère que les repos à l'unité ici